### PR TITLE
PXB-3104 Release notes for Percona XtraBackup 8.0.34-29

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/8.0/8.0.34-29.0.md
	modified:   docs/release-notes/8.0/8.0.34-29.0.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/8.0/8.0.34-29.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Percona XtraBackup 8.0.34-29 (2023-08-07)
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becames default compression method for `--compress` option.
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.34-29.0.md]: the default compression method for the `--compress` option.

File: docs/release-notes/8.0/8.0.34-29.0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Improvements
+
+* [PXB-3007](https://jira.percona.com/browse/PXB-3007): The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and is default compression method for `--compress` option.
+
+## Bug fix
+
+* [PXB-3073](https://jira.percona.com/browse/PXB-3073): The `xbcloud` got stuck while uploading the backup with 1 million tables.
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.34-29.0.md]: An unresponsive host may cause `xbcloud` to become stuck. Added a `--timeout` option to `xbcloud`

**Review by patrickbirch — APPROVED** [review]
Made a few changes to wording

File: docs/release-notes/8.0/8.0.34-29.0.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.34-29.0.md]: done

File: docs/release-notes/8.0/8.0.34-29.0.md (hunk 2 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.34-29.0.md]: done

File: docs/release-notes/8.0/8.0.34-29.0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Release highlights
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the `--compress` option.
+
+## Improvements
+
+* [PXB-3007](https://jira.percona.com/browse/PXB-3007): The [ZSTD compression method]() is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the `--compress` option.
```
Comment (patrickbirch) [inline docs/release-notes/8.0/8.0.34-29.0.md]: add a link to the compress option
create-compressed-backup.md

Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.34-29.0.md]: done

File: docs/release-notes/8.0/8.0.34-29.0.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+Get started with [Quickstart Guide for Percona XtraBackup](quickstart-overview.md).
+
+[Percona XtraBackup (PXB)](https://www.percona.com/software/mysql-database/percona-xtrabackup) is a 100% open source backup solution for all versions of Percona Server for MySQL and MySQL® that performs online non-blocking, tightly compressed, highly secure full backups on transactional systems. Maintain fully available applications during planned maintenance windows with Percona XtraBackup.
+
+## Release highlights
+
+This release merges the MySQL 8.0.34 code base. Percona has implemented a two-stage release process for each version. The first release primarily ensures compatibility with the latest MySQL version, to help those customers who need an updated version of Percona XtraBackup as soon as possible. The second release contains additional bug fixes and any improvements or new features.
+
+The [ZSTD compression method](..//..//create-compressed-backup.md) is moved to [General Availability](..//..//glossary.md#general-availability-ga) and becomes the default compression method for the [`--compress`](..//..//create-compressed-backup.md) option.
```
Comment (altmannmarcelo) [inline docs/release-notes/8.0/8.0.34-29.0.md]: Add a heading `### Compression`
Keep the note about ZSTD moving to default
Add a note that quicklz has been removed from --backup but still an option to --decompress older backups.

**Review by altmannmarcelo — CHANGES_REQUESTED** [review]
We added a few new tickets to this release:

https://jira.percona.com/browse/PXB-3088
https://jira.percona.com/browse/PXB-3079
https://jira.percona.com/browse/PXB-2860

File: docs/release-notes/8.0/8.0.34-29.0.md (hunk 4 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/8.0/8.0.34-29.0.md]: updated

**Comment by alina-derkach-oaza** [comment]
> We added a few new tickets to this release:
> 
> https://jira.percona.com/browse/PXB-3088 https://jira.percona.com/browse/PXB-3079 https://jira.percona.com/browse/PXB-2860

Added

**Pull request merged**